python/paddle/base/executor.py

```diff
@@ -1259,7 +1259,7 @@ def _new_ir_feed_data(self, program, feed, scope):
                 new_ir_check_feed_shape_type(
                     cur_feed, feed_target_name, var_shape, var_type
                 )
-                # the last arg of set_feed_variable has no effect in new ir, we pass 0 by default.
+                # the last arg of set_feed_variable has no effect in new pir, we pass 0 by default.
```

Comment:
```suggestion
                # the last arg of set_feed_variable has no effect in pir, we pass 0 by default.
```

唔，另外我觉得这种非变量的地方是不是大写会好些？因为本身是个缩写